3-cloud-computing/3.0.1-Cloud Service Ecosystems.md

```diff
@@ -0,0 +1,87 @@
+# 3.0.1 Cloud Service Ecosystems
+
+![](./img3/cloudservices.png )
+
+There are many different types of cloud service ecosystems all delivering varying levels of management for applications, data and
+their underlying infrastructure. They are as follows:
+
+## Self-Managed
```

Comment:
I would add in this section that it is not a cloud service but included to show the increasing level of outsourcing.  Want to include co-location in this section?

Reply:
good comment - resolved